stl/inc/ranges

```diff
@@ -231,7 +231,7 @@ namespace ranges {
         ~_Cached_position() = default;
 
         // Offsets are oblivious to copying, so cache values _do_ propagate via copying.
-        _Cached_position(const _Cached_position&) = default;
+        _Cached_position(const _Cached_position&)            = default;

```

Comment:
Gack. I forgot to revert the clang-format cleanups in this header. (Also in `<vector>`.)